script/constructron.lua

```diff
@@ -482,9 +482,36 @@ ctron.conditions = {
         local trash_inventory = constructron.get_inventory(defines.inventory.spider_trash)
         if trash_inventory then
             local trash_items = trash_inventory.get_contents()
-            if next(trash_items) then
+            local item = next(trash_items)
+
+            if item then
                 logistic_condition = false
             end
+            
+            local logistic_network = ctron.get_closest_service_station(constructron).logistic_network

```

Comment:
I used `get_closest_service_station` here because job.request_station does not exist during action:clear_items